client/src/pages/Apply/Apply.js

```diff
@@ -153,7 +133,7 @@ const Apply = ({ navigation }) => {
   }
 
   const onSubmit = async () => {
-    if (validate() === true) {
+    if (areRequiredFieldsFilled === true) {
```

Comment:
No need to check if `areRequiredFieldsFilled` is true since `onSubmit()` won't be called unless the button is enabled, and if the button is enabled, then `areRequiredFieldsFilled` will always be true.

Reply:
This makes sense, thanks.